jwst/regtest/test_miri_lrs_slitless.py

```diff
@@ -83,7 +86,8 @@ def test_miri_lrs_slitless_tso1(run_tso1_pipeline, rtdata_module, fitsdiff_defau
 
 
 @pytest.mark.bigdata
-@pytest.mark.parametrize("step_suffix", ["assign_wcs", "srctype", "flat_field", "calints", "x1dints"])
+@pytest.mark.parametrize("step_suffix", ["assign_wcs", "srctype", "flat_field",
+                                         "photom", "pixel_replace", "calints", "x1dints"])
```

Comment:
don't need photom, because you've got calints

Reply:
ok